tests/unit/test_time_calculations.py

```
@@ -0,0 +1,193 @@
+    def test_early_1_minute(self):
+        """Test service running 1 minute early (singular)"""
+        # estTime=16:01, arrivalTime=16:00 -> 1 min early
+        has_problem, message = plugin.delayCalc("16:01", "16:00")
+        assert has_problem is True
+        assert message == "1 min early"
+
+    def test_midnight_crossing_delay(self):
+        """Test delay calculation crossing midnight"""
+        # This is a known limitation - midnight crossing might not work correctly
+        # Documenting expected behavior
+        has_problem, message = plugin.delayCalc("23:50", "00:10")
```

Ruff (0.14.14)

Unpacked variable `message` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)